### ISSUE TITLE
Add coordinates on the four sides

### ISSUE THREAD

**Issue by ReneCareenium** [julianandrews/sgf-parse#17]
For full board diagrams, especially if a user wants to look for the coordinates of a particular spot, having coordinates on the four sides of the board is very helpful. Could this be an option? Thanks!

**Comment by julianandrews** [comment]
This looks like a good idea! 

Also, this looks like an issue in [sgf-render](https://github.com/julianandrews/sgf-render) (the parser doesn't know anything about board coordinate labels). I've opened up an issue over there to address this. I'll close this issue.

My logic for rendering the labels right now is unfortunately a bit tangled up with the board rendering logic, so it's going to take a bit of a rethink/refactor to do this gracefully, but something like this has been in the back of my head for a while. I added a bunch of tests a couple weeks ago, specifically to make this sort of refactor easier/safer to do, so next time I've got an afternoon to work on this, I'll see about doing this!
